hail/src/main/scala/is/hail/cxx/EmitTriplet.scala

```diff
@@ -113,6 +113,10 @@ abstract class NDArrayLoopEmitter(
 
   def outputElement(idxVars: Seq[Variable]): Code
 
+  def foldWithStrides(idxs: Seq[Variable], strides: Code): Code = {
```

Comment:
This name doesn't really tell me what the function does. How about `linearize_multiindex` or something?